.zombienet-tests/0001-custom.js

```diff
@@ -0,0 +1,20 @@
+const polkadotApi = require("@polkadot/api")
+
+async function connect(customChainSpec, types) {
+  const substrateConnect = await import("@substrate/connect")
+  const provider = new substrateConnect.ScProvider(customChainSpec)
+  const api = new polkadotApi.ApiPromise({ provider, types })
+  return api
+}
+
+async function run(nodeName, networkInfo) {
+  const { userDefinedTypes } = networkInfo.nodesByName[nodeName]
+  const customChainSpec = require(networkInfo.chainSpecPath)
```

Comment:
Why not move this in `connect` and pass the `networkInfo` to `connect`?

Reply:
Addressed on new PR